src/components/toolbar/import-buttons/import-buttons.component.tsx

```diff
@@ -28,11 +28,14 @@ export function ImportButtonsComponent(): JSX.Element {
         type: ScheduleDataActionType.ADD_NEW,
         payload: scheduleModel,
       } as ActionModel<ScheduleDataModel>);
-      scheduleDipatcher({
-        type: ScheduleErrorActionType.UPDATE,
-        payload: scheduleErrors,
-      } as ActionModel<ScheduleErrorModel[]>);
+    } else {
```

Comment:
Potrzeba będzie zamienić ten warunek co jest w if (26 linijka) na jakąś customową flagę, która pozwala sprawdzić czy grafik jest wczytany poprawnie.  
I ustawić to w catch.
![image](https://user-images.githubusercontent.com/21079319/98966580-bc5cd300-250b-11eb-89f8-008206d0d9e1.png)

Bo teraz wchodzimy w elese w momencie odpalenia aplikacji

![image](https://user-images.githubusercontent.com/21079319/98968873-8ec55900-250e-11eb-9493-0ee2e3a214b3.png)

PS. display message - to `console.log` który dodałem w tym elsie